packages/bpk-stylesheets/index.scss

```diff
@@ -16,65 +16,69 @@
  * limitations under the License.
  */
 
-@import '~normalize.css';
-@import '../bpk-mixins/index.scss';
+@layer base, component, utility;
 
-// stylelint-disable-next-line selector-max-type
-html {
-  font-size: $bpk-font-size-root;
-  box-sizing: border-box;
-}
+@layer base {
```

Comment:
Might be better to have normalize in its own first 'normalize' or 'reset' layer as that seems to be the tradition in learning materials I've seen